core/src/impl/Kokkos_Error.hpp

```diff
@@ -218,31 +219,43 @@ KOKKOS_IMPL_ABORT_NORETURN KOKKOS_INLINE_FUNCTION void abort(
 
 #if !defined(NDEBUG) || defined(KOKKOS_ENFORCE_CONTRACTS) || \
     defined(KOKKOS_ENABLE_DEBUG)
-#define KOKKOS_EXPECTS(...)                                               \
-  {                                                                       \
-    if (!bool(__VA_ARGS__)) {                                             \
-      ::Kokkos::abort(                                                    \
-          "Kokkos contract violation:\n  "                                \
-          "  Expected precondition `" #__VA_ARGS__ "` evaluated false."); \
-    }                                                                     \
+#define KOKKOS_STRINGIFY(x) #x
+#define KOKKOS_TOSTRING(x) KOKKOS_STRINGIFY(x)
+#define KOKKOS_EXPECTS(...)                                          \
+  {                                                                  \
+    if (!bool(__VA_ARGS__)) {                                        \
+      ::Kokkos::abort(                                               \
+          "Kokkos contract violation:\n  "                           \
+          "  Expected precondition `" #__VA_ARGS__                   \
```

Comment:
Line break look weird

Reply:
I can turn off clang-forma if that bothers you.

Reply:
Don't worry about it (unless someone else asks for it)